docs/diagrams/ChessMasterSequence.puml

```diff
@@ -0,0 +1,56 @@
+@startuml ChessMasterSequence
+
+actor User as user #BlanchedAlmond
+participant ":UI" as ui #IndianRed
+participant ":ChessMaster" as chessMaster #LightPink
+participant ":Game" as game #Orange
+participant ":Storage" as storage #DeepSkyBlue
+
+user -> chessMaster : start
+activate chessMaster
+
+chessMaster -> storage : loadBoard()
+activate storage
+
+storage --> chessMaster : prevBoard:ChessBoard
+deactivate storage
+
+alt [. Previous board exists]
+
+    chessMaster -> ui : shouldStartNewGame()
+    activate ui
+    user -> ui : "yes/no"
```

Comment:
Is this supposed to be the condition? Because its meaning kinda overlaps with Previous Board Exist, so maybe you should just use one of them